cmd/fdsn-quake-consumer/Dockerfile

```diff
@@ -1,26 +0,0 @@
-ARG BUILDER_IMAGE=ghcr.io/geonet/base-images/go:1.16
-
-FROM ${BUILDER_IMAGE} as builder
-# Obtain ca-cert and tzdata, which we will add to the container
-# Git commit SHA
-ARG GIT_COMMIT_SHA
-ARG BUILD
-ADD ./ /repo
-WORKDIR /repo
-
-# Set a bunch of go env flags
-ENV GOBIN /repo/gobin
-ENV GOPATH /usr/src/go
-ENV GOFLAGS -mod=vendor
-ENV GOOS linux
-ENV GOARCH amd64
-ENV CGO_ENABLED 0
-RUN go install -a -ldflags "-X main.Prefix=${BUILD}/${GIT_COMMIT_SHA}" /repo/cmd/fdsn-quake-consumer
-FROM ghcr.io/geonet/base-images/alpine-xslt:3.18
```

Comment:
@junghao, was there a reason why the quake consumer image has xslt installed?

(I rejigged it to use a new common base image)

Reply:
Yes it's necessary. (And my previous PR dropped it so I undo it).